.github/workflows/quicktest_runner.yaml

```diff
@@ -0,0 +1,154 @@
+name: quicktest-runner
+
+# File: quicktest_runner.yaml
+# Author: Taha Abdullah
+# Created on: 2023-07-10
+# Functionality: This workflow runs FastSurfer on MRI data and runs pytest to check if the results are acceptable. It
+#  also checks if the FastSurfer environment and output already exist, and if not, it creates them.
+# Usage: This workflow is triggered on a pull request to the dev and main branch. It can also be triggered manually
+#  with workflow-dispatch.
+# Expected/Used Environment Variables:
+#  - MAMBAPATH: Path to the micromamba binary.
+#  - MAMBAROOT: Root path for micromamba.
+#  - RUNNER_FS_OUTPUT: Path to the directory where FastSurfer output is stored.
+#  - RUNNER_FS_MRI_DATA: Path to the directory where MRI data is stored.
+#  - FREESURFER_HOME: Path to the freesurfer directory.
+#  - FS_LICENSE: Path to the FreeSurfer license file.
+
+on:
+  pull_request:
+    branches:
+      - dev
+      - stable
+  workflow_dispatch:
+
+jobs:
+  # Checkout repo
+  checkout:
+    runs-on: self-hosted
+    steps:
+    - uses: actions/checkout@v2
+
+  # Check if fastsurfer output already exists, if so, skip run-fastsurfer and run pytest directly
+  check-output:
+    runs-on: self-hosted
+    needs: checkout
+    steps:
+    - name: Check FastSurfer Output Directory
+      run: |
+        if [ -d "$RUNNER_FS_OUTPUT/subject1" ]; then
+          echo "FastSurfer output directory for subject1 exists. Finishing job..."
+          echo "HAS_FASTSURFER=false" >> $GITHUB_OUTPUT
+          exit 0
+        else
+          echo "FastSurfer output directory for subject1 does not exist. Running FastSurfer..."
+          echo "HAS_FASTSURFER=true" >> $GITHUB_OUTPUT
+          exit 1
+        fi
```

Comment:
I dont think this is "best" ... subject ids (subject1) should not be hard coded. Also, we do not wont the runs just sometimes, we want them always.